hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/tools/federation/RouterAdmin.java

```diff
@@ -1035,6 +1054,65 @@ private boolean updateQuota(String mount, long nsQuota, long ssQuota)
         .updateMountTableEntry(updateRequest);
     return updateResponse.getStatus();
   }
+  
+  /**
+   * initViewFsToMountTable.
+   * @param clusterName The specified cluster to initialize.
+   * @return If the quota was updated.
+   * @throws IOException Error adding the mount point.
+   */
+  public boolean initViewFsToMountTable(String clusterName)
+      throws IOException {
+    // fs.viewfs.mounttable.ClusterX.link./data
+    final String mountTablePrefix =
+        Constants.CONFIG_VIEWFS_PREFIX + "." + clusterName + "." +
+            Constants.CONFIG_VIEWFS_LINK + "./";
+    Map<String, String> viewFsMap = getConf().getValByRegex(mountTablePrefix);
+    if (viewFsMap.isEmpty()) {
+      System.out.println("There is no ViewFs mapping to initialize.");
+      return true;
+    }
+    for (Entry<String, String> entry : viewFsMap.entrySet()) {
+      Path path = new Path(entry.getValue());
+      DestinationOrder order = DestinationOrder.HASH;
+      String[] mount = entry.getKey().split(
```

Comment:
Would it be cleaner to use do a regex or something better than splitting blindly?